pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -0,0 +1,71 @@
+package v1alpha1
+
+/*
+Copyright 2019 The Tekton Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"github.com/knative/pkg/apis"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Check that EventListener may be validated and defaulted.
+var _ apis.Validatable = (*EventListener)(nil)
+
+// +k8s:deepcopy-gen=true
+type EventListenerSpec struct {
+	TriggerBindingRefs []TriggerBindingRef `json:"triggerbindingrefs,omitempty"`
+}
+
+// +k8s:deepcopy-gen=true
+type TriggerBindingRef struct {
+	Name       string `json:"name,omitempty"`
+	Namespace  string `json:"namespace,omitempty"`
```

Comment:
This raises a question that we haven't discussed yet: are there any qualms with allowing the `EventListener` to use `TriggerBindings` from other namespaces? Currently, I don't think that `PipelineRef` or `TaskRef` allows the use of cross-namespace `Pipelines` or `Tasks` respectively.

Reply:
hm not sure! sounds like a discussion we could start in the working group that would affect both `pipelines` and `triggers`?

Reply:
I am also not sure. My use-case will keep everything namespaced together so it is not something I feel strongly about.